### DISCUSSION TITLE
Docker Desktop - licence type

### DISCUSSION THREAD

**Discussion by doro-s** [opensafely/documentation#561] [Installing OpenSAFELY] [answered]
Hi, I'm trying to determine whether we can use the free (personal) version of Docker Desktop, or whether we need the Pro or Team licence. The free version is public repositories only, and has a limit on number of pulls/hour - this would still work for OpenSAFELY? But it also doesn't have GitHub integration which I suppose we need? 
See: https://www.docker.com/pricing  

Am I right that we require at least a Pro licence?

**Comment by evansd** [comment, marked as answer]
Hi, there's no technical requirement to use the Pro or Team version of Docker Desktop with OpenSAFELY; the free version will work absolutely fine.

The only question is whether you qualify under the terms of the free license. For many of our users, this is the relevant part of the Docker [FAQ](https://www.docker.com/pricing/faq):

> I am a researcher at a university (or another not-for-profit institution); do I or my research assistants need to purchase a Pro, Team, or Business subscription to use Docker Desktop?
>
> No. You and your assistants may use Docker Desktop for free with a Docker Personal subscription.

**Reply by doro-s** [reply]
Great, thanks a lot!
